pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -226,9 +229,10 @@ func (c *ClusterController) getCephHosts(namespace string) ([]string, error) {
 			}
 		}
 		cephPodCount[app] = len(podList.Items)
+		fmt.Fprintf(&b, "%s: %d. ", app, cephPodCount[app])
```

Comment:
Is `cephPodCount` even needed anymore?
```suggestion
		fmt.Fprintf(&b, "%s: %d. ", app, len(podList.Items))
```

Reply:
Good catch.